charts/argo-cd/templates/argocd-configs/argocd-repo-server-tls.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.repoServer.enabled .Values.repoServer.certificateSecret.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-repo-server-server-tls
```

Comment:
```suggestion
  name: argocd-repo-server-tls
```

Reply:
Oops completely missed that. Thanks for the catch. Fixed